frontera/settings/__init__.py

```diff
@@ -38,7 +56,7 @@ def __setattr__(self, name, value):
             self.__dict__[name] = value
 
     def add_module(self, module):
-        if isinstance(module, Settings):
+        if isinstance(module, BaseSettings):
```

Comment:
That's insane! Why would we or someone else need to initialize Settings class object from Settings class object? What if we remove it?